lib/phoenix_component/declarative.ex

```diff
@@ -828,10 +828,10 @@ defmodule Phoenix.Component.Declarative do
           build_attr_values_or_examples(attr)
         ]
       end,
-      if Enum.any?(attrs, &(&1.type == :global)) do
-        "\n* Global attributes are accepted."
-      else
-        ""
+      # global always goes at the end
+      case Enum.find(attrs, &(&1.type === :global)) do
+        nil -> []
+        attr -> build_attr_doc_and_default(attr, "  ")
       end
```

Comment:
Is the inline comment too much? I know code should be self-documenting, and a new test case ensures global goes last, so I can remove that part.